docs/UserGuide.md

```diff
@@ -141,17 +141,18 @@ Format: `delproj INDEX`
 ```yaml
 Note: 
 * The index must be a positive integer 1, 2, 3, …​
-* Staff list will not display any staffs. You will have to view a project to see the staff of that project
+* Staff list will not display any staff. You will have to view a project to see the staff of that project
 ```
 
 Examples:
 * `list` followed by `delproj 2` deletes the 2nd project in Project list.
-* `findproj 2103T TP` followed by `delproj 1` deletes the 1st project in the results of the `find` command.
+* `findproj 2103T TP` followed by `delproj 1` deletes the 1st project in the results of the `findproj` command.
```

Comment:
I think we should change our example commands to be something like Merger with Tiktok so that more relevant to our target audience